.github/workflows/test_valgrind.yml

```diff
@@ -8,6 +8,8 @@ on:
     branches:
       - master
   workflow_dispatch:
+  schedule:
+    - cron:  '48 4 * * *'
```

Comment:
to track the update that triggered future issues?

Reply:
:heavy_check_mark: